lib/chunkers/markdown_splitter.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Chunkers
+  class MarkdownSplitter < TextSplitter
+    MARKDOWN_DELIMITERS = [
+      "\n## ",
+      "\n### ",
+      "\n#### ",
+      "\n##### ",
+      "\n###### ",
+      "```\n\n",
+      "\n\n***\n\n",
+      "\n\n---\n\n",
+      "\n\n___\n\n",
+      "\n\n",
+      "\n",
+      " ",
+      ""
+    ].freeze
+
+    def initialize(**kwargs)
+      super(delimeters: MARKDOWN_DELIMITERS, **kwargs)
```

Comment:
Is this supposed to be `delimeter: MARKDOWN_DELIMITERS`?

Reply:
Yep. Good catch!